src/main/java/at/hannibal2/skyhanni/config/storage/ProfileSpecificStorage.java

```diff
@@ -769,7 +781,52 @@ public static class RabbitData {
         @Expose
         public long millisInCf = 0;
 
+        @Expose
+        public int rabbitTheFishFinds = 0;
+
+        public static class LeaderboardPosition {
+            @Expose
+            public int position;
+
+            @Expose
+            public double percentile;
+
+            public LeaderboardPosition(int position, double percentile) {
+                this.position = position;
+                this.percentile = percentile;
+            }
+
+            @Override
+            public int hashCode() {
+                return Objects.hash(position, percentile);
+            }
+        }
+
+        @Expose
+        public LeaderboardPosition initialLeaderboardPosition = new LeaderboardPosition(-1, -1.0);
+
+        @Expose
+        public LeaderboardPosition finalLeaderboardPosition = new LeaderboardPosition(-1, -1.0);
+
+        @Expose
+        public long lastLbUpdateMarkMillis = Long.MAX_VALUE;
```

Comment:
```suggestion
        public SimpleTimeMark lastLbUpdate = SimpleTimeMarkFarPast();
```
this will need a couple changes in the HoppityEventSummary class